app/views/catalog/_index_gallery_collection_wrapper.html.erb

```diff
@@ -0,0 +1,11 @@
+<%# OVERRIDE Hyrax 3.4.1: make collection thumbnail a link so it matches works %>
+<div class="document col-xs-6 col-md-3">
+  <div class="thumbnail">
+  <% debugger %>
```

Comment:
Oops! spotted a debugger.

Reply:
I don't know how that happened because that isn't in my code and it doesn't show any changes. :/